recipe/meta.yaml

```diff
@@ -46,3 +46,4 @@ about:
 extra:
   recipe-maintainers:
     - jseabold
+    - beckermr
```

Comment:
The changes in this file are by me too.

Reply:
The rest of the changes in the PR are due to conda-forge rerendering the feedstock to update the build.